package.json

```diff
@@ -4,58 +4,56 @@
   "description": "Vela UI",
   "author": "Vela Team",
   "engines": {
-    "node": ">=12.14.1"
+    "node": ">=14.16.0"
   },
   "dependencies": {
-    "@fullhuman/postcss-purgecss": "3.1.3",
-    "@types/clipboard": "2.0.1",
-    "@types/parcel-env": "0.0.0",
-    "autoprefixer": "9.8.6",
-    "clipboard": "2.0.6",
-    "elm": "0.19.1-4",
-    "node-elm-compiler": "5.0.5",
-    "parcel-bundler": "1.12.4",
-    "parcel-plugin-static-files-copy": "2.5.0",
-    "rimraf": "3.0.2",
-    "sass": "1.32.5",
-    "typescript": "4.1.3"
+    "clipboard": "2.0.8"
   },
   "devDependencies": {
-    "axe-core": "4.1.1",
+    "@fullhuman/postcss-purgecss": "4.0.2",
+    "@parcel/transformer-elm": "2.0.0-nightly.2249",
+    "@parcel/transformer-sass": "2.0.0-nightly.627",
+    "axe-core": "4.1.3",
     "cypress": "5.6.0",
-    "cypress-axe": "0.12.0",
-    "elm-hot": "1.1.6",
+    "cypress-axe": "0.12.2",
+    "elm": "0.19.1-5",
+    "make-dir-cli": "2.0.0",
+    "ncp": "2.0.0",
```

Comment:
mainly added to deal with copying static image/favicon assets for builds. we used a parcel plugin previously and while the plugin has an update to work with Parcel v2, that plugin would be the only reason that would cause us to have a parcel configuration file at this time.

the alternative would be to use the `import .. from "url:./.."` syntax in Parcel v2 but then you have asset hashing to deal with, etc